hooks/msghooks.py

```diff
@@ -115,6 +115,11 @@ def receive_message():
                           /,/
                           |/`)
                           'c=,"""
+        if re.match(r"(make|pls make|plz make|please make) (me | meh) a (cookie|cokie)", without_ping):
```

Comment:
This regex looks not epic. Maybe try `(pl(s|z|ease) )?make meh? a coo?kie`?

Reply:
It'd be nice if it also gave you things other than cookies (biscuits, ice cream, money, coke, etc.)

Reply:
cookie